Changes

```diff
@@ -14,6 +14,10 @@ Working version
   (Stephen Dolan, review by Leo White, Gabriel Scherer and Jacques-Henri
    Jourdan)
 
+- #10102: Take PROFINFO_WIDTH=0 if WITH_PROFINFO is not defined (technically
```

Comment:
This confused me a bit, because `PROFINFO_WIDTH` is (normally) already 0 in this case.  What about:
```suggestion
- #10102: Ignore PROFINFO_WIDTH if WITH_PROFINFO is not defined (technically
```

Reply:
That's much clearer, yes!